spec/rubocop/cop/performance/sort_with_block_spec.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Performance::SortWithBlock do
+  subject(:cop) { described_class.new }
+
+  it 'autocorrects array.sort { |a, b| a.foo <=> b.foo }' do
```

Comment:
Typically we also have some standalone tests testing simply that offenses are being registered/not registered. I'd definitely add some negative test cases.